ovs/extensions/hypervisor/apis/kvm/sdk.py

```diff
@@ -387,6 +387,55 @@ def clone_vm(self, vmid, name, disks, mountpoint):
         return self.create_vm_from_template(name, source_vm, disks, mountpoint)
 
 
+    def create_volume(self, location, size):
+        """
+        Create volume using truncate
+        @param location: Location on filesystem
```

Comment:
Please use ```:param``` style comments, were changing everything to that format and want to keep it as consistent as possible.